src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.ComponentModel;
```

Comment:
probably should be deleted

Reply:
This is for the `Win32Exception` that is thrown by the `pipe2` method.